### ISSUE TITLE
Removing Iris, FaceMesh from the Holistic model?

### ISSUE THREAD

**Issue by midopooler** [homuler/MediaPipeUnityPlugin#189]
Hello @homuler , How do we remove Iris, FaceMesh from the Holistic model. I am asking this because in some cases the Iris, FaceMesh models gets heavy, and I don't need them in my project.

I tried commenting out the Facemesh graph from the 3 main scripts of the holistic model, namely `HolisticGraph`, `HolisticValue` and `HolisticAnnotationController` . But upon running I get null reference which I was not able to debug it in a standard way.  
Here are the logs,  
![image](https://user-images.githubusercontent.com/43686606/120496215-05f30980-c3db-11eb-815a-c51a21fa04ff.png)

Can you please suggest a better way of doing this?

**Comment by homuler** [comment]
You can disable iris detections by setting `false` to `detectIris`.
https://github.com/homuler/MediaPipeUnityPlugin/blob/8c171d0eb2d16140ea94511d3dd6c94cf3792804/Assets/Mediapipe/Samples/Graphs/Holistic/Scripts/HolisticGraph.cs#L11

Disabling FaceMesh is not so straightforward, because `HolisticLandmarkGpu`, which `HolisticGraph` [uses internally](https://github.com/homuler/MediaPipeUnityPlugin/blob/8c171d0eb2d16140ea94511d3dd6c94cf3792804/Assets/Mediapipe/Samples/Graphs/Holistic/Resources/holistic_desktop_gpu.txt#L95
), does not have such an option.

There are at least 2 options.
1. You can inline `HolisticLandmarkGpu` except for [unnecessary nodes](https://github.com/google/mediapipe/blob/017c1dc7eaf2d45ca95769194d9c83a86a82876d/mediapipe/modules/holistic_landmark/holistic_landmark_gpu.pbtxt#L108-L114).
1. You can define you custom calculators (say `PoseLandmarksWithHandsGpu`) and use it from `HolisticGraph` (cf. https://github.com/google/mediapipe/blob/017c1dc7eaf2d45ca95769194d9c83a86a82876d/mediapipe/modules/holistic_landmark/BUILD#L241-L253).

**Comment by midopooler** [comment]
Hi @homuler , what you have described is really interesting way (by making a custom calculator) 

But I was able to do my way as well, which I was telling in the first comment, by editing `HolisticGraph`, `HolisticValue` and `HolisticAnnotationController` . The only I did was to comment out all the lines of code where the face mesh was used. 

Where I got stuck in the first comment was a stupid error, I mistakenly used another prefab in the inspector. 
Anyways, Let say If I want to add some other graph (for example lets take face detection instead of face mesh) to add in the holistic model, How do we do it?
Is it by just adding relevant code in all the three scripts I mentioned or something else?

I tried adding faceDetection with this method as shown in the images, Am I going in right way?
![image](https://user-images.githubusercontent.com/43686606/120602583-ca068580-c468-11eb-82cf-73f34daf4183.png)
![image](https://user-images.githubusercontent.com/43686606/120602642-dab6fb80-c468-11eb-8553-b0311a1de545.png)
![image](https://user-images.githubusercontent.com/43686606/120602722-eb677180-c468-11eb-803f-505c3728b64f.png)
